tests/e2e/runtime/products.spec.ts

```diff
@@ -1,42 +1,35 @@
-import * as puppeteer from "puppeteer";
-import { expect } from "chai";
-import { Utils } from "../../utils";
-import { BrowserLaunchOptions } from "../../constants";
-import { Server } from "http";
-import { Products } from "../../mocks/collection/products";
 import { Product } from "../../mocks/collection/product";
 import { ProductseWidget } from "../maps/products";
+import { test, expect } from '../playwright-test';
+import { Templating } from "../../templating";
 
-describe("Products page", async () => {
-    let config;
-    let browser: puppeteer.Browser;
-    let server: Server;
-    
-    before(async () => {
-        config = await Utils.getConfig();
-        browser = await puppeteer.launch(BrowserLaunchOptions);
-    });
-    after(async () => {
-        await browser.close();
-        Utils.closeServer(server);
-    });
-
-    it("User can see producst on the page", async () => {
-        var products = new Products();
-        products.addProduct(Product.getStartedProduct());
-        products.addProduct(Product.getUnlimitedProduct());
+test.describe("products-page", async () => {
+    test("published-products-visible-to-guests", async function ({page, configuration, cleanUp, mockedData, productService, testRunner})  { 
```

Comment:
I think what we can do to ease the devs in creating the test we can provide them with small helper to create all of mock data preparation, data population, and validation. Please consider that.

Reply:
I would like to keep the changes for the next iteration, reasons behind that: 
1 - To see the usage of the test by developers and gather more feedback
2 - To understand main bottlenecks while implementing the test